spring-boot-data-dynamodb/src/main/resources/logback.xml

```diff
@@ -0,0 +1,14 @@
+<configuration>
+
+	<appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">
+		<encoder>
+			<pattern>web - %date [%thread] %-5level %logger{36} - %message%n
+			</pattern>
+		</encoder>
+	</appender>
+
+	<root level="INFO">
+		<appender-ref ref="STDOUT" />
+	</root>
+
+</configuration>
```

Comment:
![image](https://cloud.githubusercontent.com/assets/4750426/19377367/a6e49b10-919a-11e6-99ff-b3f2cb40a254.png)

It is 4 spaces in the actual code if you open it up.  It just is formatted to look like a tab in the github UI